[Marketplace Contribution] NetskopeV2 - Content Pack Update (#33527)
demisto/content: Packs/NetskopeV2/ReleaseNotes/1_0_3.md

#### Integrations

##### Netskope (API v2)

> - %%UPDATE_RN%%

Reviewer: please fill in the note, explaining the changes you made

> - Fixed and issue caused by Netskope modifying their API to reject requests ending in a forward slash

Reviewer: ```suggestion
- Updated the API requests due to  *Netskope* modifying their API.
```
